class AuthenticationError(Exception):
    pass


class LoginError(AuthenticationError):
    pass